Public Relations Approval Process.md

```diff
@@ -0,0 +1,8 @@
+**Publication Approval Process**
+
+1.  Team member drafts content for release (this includes posters, presentations, manuscripts, and anything that will be released publicly that is branded KBase).
```

Comment:
How about if we replace "branded KBase" with "about KBase or referring to KBase in a significant way (for example, showing analyses that were performed in KBase)"?
Also: "Not only will the members of PR-Approval be vetting the docs for sensitive/inappropriate content, but they will also do some editing before any untoward typos reach the public."  -- maybe some of that elaboration should be added to #3? (Or maybe not, since this writeup of the PR approval process was intended to be succinct enough that people would actually read it!)